dcos/dcos-ui
# DCOS-57423: Quota Detail view is not scrollable

Make the Quota detail page scrollable.

Closes https://jira.mesosphere.com/browse/DCOS-57423

## Testing
1. Go to services tab.
2. Create a group with quota.
3. Go to quota tab.
4. Click on that group.
5. Make the browser window small.
6. Verify that it is scrollable.

## Trade-offs
None.

## Dependencies
None.

## Screenshots
### Before
![Снимка от 2019-08-07 10-49-41](https://user-images.githubusercontent.com/40791275/62616384-0a058b80-b918-11e9-9911-9b3d711012de.png)

### After
![Снимка от 2019-08-07 13-26-04](https://user-images.githubusercontent.com/40791275/62616391-0c67e580-b918-11e9-9cc6-2f6a26149edf.png)
